linux arm64 build error

https://travis-ci.com/github/OpenIoTHub/server-go/jobs/327449253#L649
   ⨯ release failed after 384.08s error=failed to build for linux_arm64: # github.com/klauspost/reedsolomon
../../../../pkg/mod/github.com/klauspost/reedsolomon@v1.9.5/reedsolomon.go:464:4: r.codeSomeShardsAvx512P undefined (type reedSolomon has no field or method codeSomeShardsAvx512P)

--- a/galois_arm64.go
+++ b/galois_arm64.go
@@ -47,6 +47,3 @@ func sliceXor(in, out []byte, sse2 bool) {
 		out[n] ^= input
 	}
 }
-
-func (r reedSolomon) codeSomeShardsAvx512(matrixRows, inputs, outputs [][]byte, outputCount, byteCount int) {
-}

--- a/galois_noasm.go
+++ b/galois_noasm.go
@@ -32,11 +32,3 @@ func sliceXor(in, out []byte, sse2 bool) {
 func init() {
 	defaultOptions.useAVX512 = false
 }
-
-func (r reedSolomon) codeSomeShardsAvx512(matrixRows, inputs, outputs [][]byte, outputCount, byteCount int) {
-	panic("codeSomeShardsAvx512 should not be called if built without asm")
-}
-
-func (r reedSolomon) codeSomeShardsAvx512P(matrixRows, inputs, outputs [][]byte, outputCount, byteCount int) {
-	panic("codeSomeShardsAvx512P should not be called if built without asm")
-}

--- a/galois_notamd64.go
+++ b/galois_notamd64.go
@@ -1,0 +1,13 @@
+//+build !amd64 noasm appengine gccgo
+
+// Copyright 2020, Klaus Post, see LICENSE for details.
+
+package reedsolomon
+
+func (r reedSolomon) codeSomeShardsAvx512(matrixRows, inputs, outputs [][]byte, outputCount, byteCount int) {
+	panic("codeSomeShardsAvx512 should not be called if built without asm")
+}
+
+func (r reedSolomon) codeSomeShardsAvx512P(matrixRows, inputs, outputs [][]byte, outputCount, byteCount int) {
+	panic("codeSomeShardsAvx512P should not be called if built without asm")
+}

--- a/galois_ppc64le.go
+++ b/galois_ppc64le.go
@@ -65,6 +65,3 @@ func sliceXor(in, out []byte, sse2 bool) {
 		out[n] ^= input
 	}
 }
-
-func (r reedSolomon) codeSomeShardsAvx512(matrixRows, inputs, outputs [][]byte, outputCount, byteCount int) {
-}

